local-volume/provisioner/pkg/common/common.go

```diff
@@ -66,6 +68,8 @@ type UserConfig struct {
 	Node *v1.Node
 	// key = storageclass, value = mount configuration for the storageclass
 	DiscoveryMap map[string]MountConfig
+	// Labels that should be applied on PVs created by the provisioner
```

Comment:
"Labels and their values that are added to PVs created by the provisioner."

Reply:
Fixed.